vlib/v/checker/tests/is_type_not_exist.out

```diff
@@ -1,12 +1,12 @@
-vlib/v/checker/tests/is_type_not_exist.vv:4:2: error: cannot use type `any_int` as type `Integer` in argument 1 to `fn_with_sum_type_param`
-    2 |
+vlib/v/checker/tests/is_type_not_exist.vv:4:26: error: invalid argument 1 to `fn_with_sum_type_param`: expected `Integer`, not `any_int`
+    2 | 
```

Comment:
not really related, but `any_int` should be replaced with `number literal` in all error messages

that's much more clear for the end user